### DISCUSSION TITLE
ScriptReference/StaticEditorFlags

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#221] [Notes]
# ScriptReference/StaticEditorFlags

https://docs.unity3d.com/ScriptReference/StaticEditorFlags.html

**Comment by mvi** [comment]
You can get and set these values using [`GameObjectUtility.SetStaticEditorFlags`](https://docs.unity3d.com/ScriptReference/GameObjectUtility.SetStaticEditorFlags.html) and [`GameObjectUtility.GetStaticEditorFlags`](https://docs.unity3d.com/ScriptReference/GameObjectUtility.GetStaticEditorFlags.html).  
Note that this enum as well as `GameObjectUtility` is part of `UnityEditor`, so you cannot use it in builds. This is because static flags are really an editor concept and don't really exist at runtime.
